packages/eth-sdk/README.md

```diff
@@ -28,8 +28,10 @@
     - [`outputPath`](#outputpath)
     - [`etherscanKey`](#etherscankey)
     - [`etherscanURLs`](#etherscanurls)
+    - [`rpc`](#rpc)
+    - [`noFollowProxies`](#nofollowproxies)
 - [Examples](#examples)
-- [Videos](#videos)
+  - [Videos](#videos)
```

Comment:
There is a Markdown All in One extension for VSCode which updates the TOC automatically.